tests/integration/expectations/operations/test_dataset.py

```
@@ -1123,3 +1124,163 @@ def test_duplicate_name_check_threshold_from_config(dataset_path):
+def test_count_lpa_boundary_marks_a_failed_geometry_fetch_as_an_error(
+    dataset_path, mocker
+):
+    """a check that could not run must be distinguishable from a real finding without
+    string-matching the message"""
+    mocker.patch(
+        "requests.get",
+        side_effect=requests.exceptions.RequestException("404 Client Error"),
+    )
+
+    with spatialite.connect(dataset_path) as conn:
+        passed, message, details = count_lpa_boundary(conn, lpa="test", expected=0)
```

Ruff (0.16.2)

[warning] Unpacked variable `message` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)